### ISSUE TITLE
Amphibious gear switch should not triger repair timer while  pontoon damaged

### ISSUE THREAD

**Issue by wlbragg** [c172p-team/c172p#1290]
I missed this in my rush to get this in 2019.1. This fixes the behavior but there is still an issue that the amphibious gear trigger a repair when they are switched when not damaged. So if say a wing is damaged and you raised or retract the gear, it gets repaired. I can't find the trigger here. I looked at the "repairing" flag but don't see that triggering a repair, only a delay on all damages and tiedown support.
Bottom line, it's not critical and we can look at it later, I simply want to get the major bug fix in 2019.1 and this finishes that part of it.
